src/app/issues-viewer/card-view/card-view.component.html

```diff
@@ -18,35 +18,7 @@
   </div>
 
   <div *ngFor="let issue of this.issues$ | async; index as i">
-    <mat-card class="card" [ngClass]="this.getIssueOpenOrCloseColorCSSClass(issue)">
-      <a class="no-underline link-grey-dark" (click)="viewIssueInBrowser(issue.id, $event)">
-        <span [matTooltip]="this.issue.updated_at">
-          <mat-card-header>
-            <mat-card-title>
-              <span class="octicon" [octicon]="this.getOcticon(issue)" [color]="this.getIssueOpenOrCloseColor(issue)"></span>
-              #{{ issue.id }}: {{ this.fitTitleText(issue.title) }}
-            </mat-card-title>
-          </mat-card-header>
-          <mat-card-content>
-            <mat-chip-list aria-label="Labels">
-              <ng-container *ngIf="issue.milestone.state">
-                <span class="octicon-milestone" octicon="milestone" color="grey" size="8"></span>
-                {{ issue.milestone.title }}
-              </ng-container>
-              <ng-container *ngFor="let label of issue.githubLabels">
-                <mat-chip
-                  *ngIf="!issues.dropdownFilter?.hiddenLabels.has(label.name)"
-                  [ngStyle]="labelService.setLabelStyle(label.color)"
-                  selected
-                >
-                  {{ label.name }}
-                </mat-chip>
-              </ng-container>
-            </mat-chip-list>
-          </mat-card-content>
-        </span>
-      </a>
-    </mat-card>
+    <app-issue-pr-card [issue]="issue" [dropdownFilter]="issues.dropdownFilter"></app-issue-pr-card>
```

Comment:
Over here, let's make it more composable instead, perhaps by declaring the IssuePrCard like this
```html
<app-issue-pr-card [issue]="issue">
  <app-issue-pr-card-header header [add the necessary props here...]></app-issue-pr-card-header>
  <app-issue-pr-card-milestone milestone [add the necessary props...]></app-issue-pr-card-milestone>
  <app-issue-pr-card-labels labels [add the necessary props here...]></app-issue-pr-card-labels>
</app-issue-pr-card>
```

Reply:
I think this is a good suggestion but I feel that the reusability of each of the element here might not be that reusable although this will certainly be more readable compared to before. I initially made this pr just to separate out the necessary elements so I can reuse this component in other pages that's why I did not really change the inner workings of the mat-card. 

I make the elements more composable to improve readability but I think I will not expose the header/milestone/labels components since this is really just a inner component for the issue-pr-card component.